addons/mass_mailing/wizard/mailing_list_add_contact_views.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<odoo>
+
+        <record model="ir.ui.view" id="view_mailing_list_add_contact_form">
+            <field name="name">mailing.list.add.contact.form</field>
+            <field name="model">mailing.list.add.contact</field>
+            <field name="arch" type="xml">
+                <form string="Send a Sample Mail">
+                    <p class="text-muted">
+                        Add contacts to mailing list
+                    </p>
+                    <group>
+                        <field name="mailing_list_id"/>
+                    </group>
+                    <footer>
+                        <button string="Add" name="action_add_contact_to_mailing_list" type="object" class="btn-primary"/>
+                        <button string="Cancel" class="btn-secondary" special="cancel" />
+                    </footer>
+                </form>
+            </field>
+        </record>
+
+        <record id="action_mailing_list_add_contact" model="ir.actions.act_window">
+            <field name="name">Add contact to a mailing list</field>
```

Comment:
Since you add several contacts at a time, maybe we should make everything plural?
- Add contact**s** to a mailing list
- action_mailing_list_add_contact**s**
- mailing.list.add.contact**s**
- ...